lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -545,6 +573,10 @@ def main():
         posixfs=dict(default=None, type='dict'),
         glusterfs=dict(default=None, type='dict'),
         fcp=dict(default=None, type='dict'),
+        wipe_after_delete=dict(type='bool', default=None),
+        backup=dict(type='bool', default=False),
```

Comment:
Do we really want to have default `False` or do we want to have default `None` and let backend decide?

Reply:
sure, will change to None